helm/hpcc/templates/roxie.yaml

```diff
@@ -219,6 +222,9 @@ spec:
 {{- end }}
       annotations:
         checksum/config: {{ $configSHA }}
+{{- if hasKey $.Values.global "metrics" }}
+{{ include "hpcc.addPrometheusScrapeAnnotations" $.Values.global.metrics | indent 8 }}
```

Comment:
indentation as per last comment

Reply:
I actually tried to follow existing convention, see L:220-222 just above

Reply:
right, it's a bit of a inconsistent mess at the moment. But we agreed that indenting is clearer, so for new or modified code we should indent blocks.